common/io/SelectServerTest.cpp

```diff
@@ -582,7 +582,7 @@ void SelectServerTest::testTimeout() {
       10,
       ola::NewSingleCallback(this, &SelectServerTest::ReentrantTimeout, m_ss));
   m_ss->RegisterSingleTimeout(
-      20,
+      40,
```

Comment:
With the logging changes below in, and my timeout, it logs the following:
common/io/SelectServerTest.cpp:135: Re-entrant timeout called, adding two single
 increment timeouts
common/io/SelectServerTest.cpp:129: Single increment timeout called
common/io/SelectServerTest.cpp:131: Timeout counter is now 1
common/io/SelectServerTest.cpp:129: Single increment timeout called
common/io/SelectServerTest.cpp:131: Timeout counter is now 2
common/io/SelectServerTest.cpp:124: Terminate called

Which looks like it's working correctly.

With that time knocked down to 35ms I get:
common/io/SelectServerTest.cpp:135: Re-entrant timeout called, adding two single
 increment timeouts
common/io/SelectServerTest.cpp:124: Terminate called
common/io/SelectServerTest.cpp:129: Single increment timeout called
common/io/SelectServerTest.cpp:131: Timeout counter is now 1
common/io/SelectServerTest.cpp:596: Timeout counter is now 1